tests/pytests/data/eis_wac_pb/eis_wac_pb_isis3.lbl

```diff
@@ -0,0 +1,57 @@
+Object = IsisCube
+    Object = Core
+        StartByte   = 65537
+        Format      = Tile
+        TileSamples = 1024
+        TileLines   = 1024
+
+        Group = Dimensions
+            Samples = 1335
+            Lines   = 4096
+            Bands   = 1
+        End_Group
+
+        Group = Pixels
+            Type       = SignedWord
+            ByteOrder  = Lsb
+            Base       = 0.0
+            Multiplier = 1.0
+        End_Group
+    End_Object
+
+    Group = Instrument
+        SpacecraftName   = "Europa Clipper"
+        InstrumentId     = "EIS WAC PB"
+        TargetName       = Didymos
+        StartTime        = 2021-03-06T04:05:27.77
+        ExposureDuration = 1.0 <seconds>
+    End_Group
+
+    Group = BandBin
+        FilterName = BLU
+        Center     = 427.5 <nm>
+        Width      = 95 <nm>
+    End_Group
```

Comment:
Are these the right values for the `BandBin`? I have different values in one of the tests within ISIS, I just wanted to double-check.
```
      Group = BandBin
        FilterName = CLEAR
        Center     = 712.5 <nm>
        Width      = 675 <nm>
      End_Group
```

Reply:
I manually changed the filters on the PB tests to check some of the filter-based logic. These are the values for the BLU filter from our ingestion template.